Maestro/subscriptions.json

```diff
@@ -299,6 +305,13 @@
         }
       }
     },
+    // Mirror corefx changes into VSO
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/corefx/master/**/*",
+      ],
+      "action": "corefx-mirror"
+    },
```

Comment:
This will only work with the master branch trigger, not any of the other branches. Looks fine as an incremental change, but make sure only to disable the master mirror build def trigger. With how Maestro currently works, you'll need a subscription entry for each branch, specifying `vsoSourceBranch`.